README.md

```diff
@@ -1,73 +1,87 @@
 
+## 🎬 Live Demo
 
-## 🎬 Live demo
+Check out the [live demo of Kitty Items](https://dark-frost-1788.on.fleek.co/),
+deployed on [IPFS](https://docs.ipfs.io/concepts/case-study-fleek/) and the Flow Testnet.
 
-### Check out the [live demo of Kitty Items](https://dark-frost-1788.on.fleek.co/), deployed on <a href="https://docs.ipfs.io/concepts/case-study-fleek/"><img height="28px" src="https://docs.ipfs.io/assets/img/logo-fleek.5aed66a3.png" /></a> and the <img height="28px" src="https://assets.website-files.com/5f6294c0c7a8cdd643b1c820/5f6294c0c7a8cda55cb1c936_Flow_Wordmark.svg" /> Testnet.
 ---
 
-# ✨ Get Started
+# ✨ Getting Started
 
-1) ### Create a Flow Testnet account <br/>
-To work on this project you'll need to create an account on the Flow Testnet:
+### 1. Install the Flow CLI
 
-- Generate your new Flow account keys using the following command from [Flow CLI](https://docs.onflow.org/flow-cli/): ```flow keys generate``` ⚠️ Make sure to           save these keys in a safe place, you'll need them later.
-- Sign up for a Testnet account here: https://testnet-faucet.onflow.org/ by filling out the "Creating an Account" form using the **public key** generated           from the previous step.
-- Click "Submit". After a short time, you'll receive an e-mail with your newly created Flow **account address**.
-    
+Before you start, install the [Flow command-line interface (CLI)](https://docs.onflow.org/flow-cli).
 
-2) ### Follow [these instructions](https://github.com/onflow/flow-cli) to install Flow (command-line tools)
+### 2. Create a Flow Testnet account
 
-## 
+You'll need a Testnet account to work on this project. Here's how to make one:
 
-<img width="22px" src="https://user-images.githubusercontent.com/674621/71187801-14e60a80-2280-11ea-94c9-e56576f76baf.png" /> Using VSCode? Before cloning this project download and install the [Cadence VSCode Extension](https://github.com/onflow/vscode-flow). 
-```
-flow cadence install-vscode-extension
-```
+- Generate a new key pair with the Flow CLI: ```flow keys generate``` (_⚠️ Make sure to save these keys in a safe place, you'll need them later._)
+- Go to the [Flow Testnet Faucet](https://testnet-faucet.onflow.org/) to create a new account. Use the **public key** from the previous step.
+- After a short time, you'll receive an email with your newly-created Flow **account address**.
 
-##
+### 3. Clone the project
 
-3) ### Clone the project
-
-```
+```sh
 git clone git@github.com:onflow/kitty-items.git
 ```
 
-Once you have the project files, you'll find setup instructions in the `README.md` files inside each folder. 
-The Cadence code is also well commented to help you understand how they function and how to use them. 
+Once you have the project files, you'll find setup instructions in the `README.md` files inside each folder.
+
+### 4. Deploy the contracts
+
+Follow the [deployer instructions]() to deploy the Kitty Items Cadence contracts to your Testnet account.
```

Comment:
What should be here? Do we leave CLI deployment as part of https://github.com/onflow/kitty-items/tree/master/kitty-items-js tutorial or should we add it here?

Reply:
Good catch, thank you! 

I meant to link this to the `kitty-items-deployer` README for now, but will replace with the CLI instructions once that branch is ready.